### PR TITLE
WIP: Feature/use asyncio loop

### PR DESCRIPTION
**Pull request by ybrs**
Remove tornado dependency. Move to asyncio subprocess

### CODE REVIEW

**Comment by s4ke** [comment]
We should bump the version. But besides this I don't see anything that needs changing right now.

**Comment by s4ke** [comment]
If it is working as is right now, should we maybe merge this PR and split the rest into a follow up PR?

**Comment by ybrs** [comment]
> If it is working as is right now, should we maybe merge this PR and split the rest into a follow up PR? While the reformatting is great, it would be easier to review if we did changes incrementally.

I'll add a few more tests and a bit cleanup, then we can merge this.

**Review by s4ke — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -13,18 +13,19 @@
     license='MIT',
     packages=['singlebeat'],
     zip_safe=True,
+    python_requires='>3.7.0',
     install_requires=[
         'tornado >= 6.0,<7.0',
```
Comment (s4ke) [inline setup.py:18]: is this still required?

**Pull request merged**